packages/firebase_storage/firebase_storage_web/pubspec.yaml

```diff
@@ -8,24 +8,23 @@ environment:
   flutter: ">=1.20.0 <2.0.0"
 
 dependencies:
-  async: ^2.4.2
-  crypto: ^2.1.5
+  async: ^2.5.0-nullsafety.3
```

Comment:
pulled from the repo, as null safe version is not visible from pub.dev. Not sure if this is correct.